config/routes.js

```diff
@@ -100,4 +100,6 @@ module.exports.routes = {
   '/test/loadSeasonFixture': 'TestController.loadSeasonFixture',
   '/test/loadMatchFixtures': 'TestController.loadMatchFixtures',
   '/test/loadFinishedGameFixtures': 'TestController.loadFinishedGameFixtures',
+
+  '/*': { controller: 'ViewController', action: 'serveIndex', skipAssets: true }
```

Comment:
In my PR I was trying to figure out how to skip assets and couldn't find the answer. This is way better than the custom middleware I was using, really nice work!